content/boilerplates/notes/1.1.4.md

```diff
@@ -0,0 +1,42 @@
+## Behavior change
+
+- Changed the default behavior for Pilot to enable fall through routes which allows workloads to access external host using the same port as any
+internal HTTP service. This behavior can be controlled by the `PILOT_ENABLE_FALLTHROUGH_ROUTE` environment variable.
```

Comment:
@howardjohn any reason why any one would want to set this `PILOT_ENABLE_FALLTHROUGH_ROUTE`  to false?

Reply:
https://xkcd.com/1172/

Alternatively, if something goes wrong and it turns out there is a bug, it causes perf regressions, etc.

There shouldn't be though, we won't have the flag in 1.2, its just as a safety switch because its a new change mid-release